BUGFIX: 3839 show warn dialog before ckeditor formatting gets lost

mhsdesign opened the pull request:
Resolves: #3839


https://github.com/user-attachments/assets/02c42bdc-4452-493e-bf74-9cf07408d0b2



**What I did**

**How I did it**

**How to verify it**

<!--
If possible, a screenshot or a gif comparing the new and old behavior would be great.
-->

mhsdesign committed:
BUGFIX: 3839 show warn dialog before ckeditor formatting gets lost

grebaldi reviewed (changes requested):
Looks good after a first quick skim, but I noticed a naming issue:

`derivation, *Derivation` should be `deviation, *Deviation`, because

|English|German|
|-|-|
|derivation|Ableitung|
|deviation|Abweichung|

mhsdesign committed:
WIP: Fix neos ckeditor outline overlay

https://github.com/neos/neos-ui/commit/95716f791b9be5c59e6de0a45d76eadd6e6dafbd

https://github.com/neos/neos-ui/commit/e551201b582d604ca494b8251ee2f67abb3ba352
